selfdrive/car/hyundai/hyundaican.py

```diff
@@ -30,9 +31,20 @@ def create_lkas11(packer, apply_steer, steer_req, cnt, enabled, lkas11, hud_aler
   }
 
   dat = packer.make_can_msg("LKAS11", 0, values)[2]
-  dat = dat[:6] + dat[7]
-  checksum = hyundai_checksum(dat)
 
+  # CRC Checksum as seen on 2019 Hyundai Santa Fe
+  if CHECKSUM[car_fingerprint] == 0:
```

Comment:
`if CHECKSUM[car_fingerprint] == CHECKSUM[CAR.SANTA_FE]:` is nicer IMO

Reply:
While that is true, it will be the opposite when multiple vehicles support that checksum.
Same with the 6 byte and 7 byte checksums.
When 20 hyundai/kia models are supported direct model definition will become really messy.